app/controllers/users_controller.rb

```diff
@@ -6,21 +6,46 @@ class UsersController < ApplicationController
 
 	def show
 		@age = current_user.age
-	    @gender = current_user.gender
-	    @weight = current_user.weight
-	    @height = current_user.height
-	    @exercise = current_user.exercise
-	    @goal = current_user.goal
-	    @budget = current_user.budget
-	    @time = current_user.time
-	    @cuisine = current_user.cuisine
-	    if @gender == 'Male'
-		    @calories = 10*@weight + 6.25*@height - 5*@age + 5
-		else
-			@calories = 10*@weight + 6.25*@height - 5*@age - 161
-		end
+	    @gender = :gender
+	    @weight = :weight
+	    @height = :height
+	    @exercise = :exercise
+	    @goal = :goal
+	    @budget = :budget
+	    @time = :time
+	    @cuisine = :cuisine
+	    @calories = calc_calories(@gender, @weight, @height, @age, @exercise, @goal)
+	end
+
```

Comment:
good idea to short variables. i think it def helps us in the long run. also should we cite where we found the calculation somewhere in the comments?

Reply:
Yeah, I added links to the places I found those calculations and numbers